packages/webpack/src/loader.ts

```diff
@@ -96,6 +98,11 @@ export default async function vocabLoader(this: LoaderContext) {
           ${renderLanguageLoader(config.devLanguage)},
           ${getAltLanguages(config)
             .map((altLanguage) => renderLanguageLoader(altLanguage))
+            .join(',')},
+          ${config.generatedLanguages
```

Comment:
I realise this predates what you've written but it feels wrong that we interrogate the user config so much here. Instead of reading the `devLanguage` and `generatedLanguages` etc, can we just iterate `Object.keys(loadedTranslation.langauges)`?